darkside.cdf

```diff
@@ -93,6 +94,7 @@ card "/starwars/SpecialEdition-Dark/t_dodobodonawieedo" "
 card "/starwars/Virtual9-Dark/t_drchellilonaaphra" "�Dr. Chelli Lona Aphra (2)\nDark Character - Alien [R1]\nSet: Set 9\nPower: 3 Ability: 3\nDeploy: 3 Forfeit: 3\nIcons: Pilot, Warrior\n\nLore: Female information broker, spy, and thief. Trade Federation.\n\nText: Adds 1 to power of anything she pilots. Deploys free to an unoccupied site. Your droids are deploy -1 here. When deployed, may search your Lost Pile and move one card there ot the top of that pile (if that card is a droid, may retrieve it into hand)."
 card "/starwars/Premiere-Dark/t_drevazan" "�Dr. Evazan (3)\nDark Character - Alien [R2]\nSet: Premiere\nPower: 1 Ability: 2\nDeploy: 2 Forfeit: 3\nIcons: Pilot, Warrior\n\nLore: Thug. Smuggler. Twisted 'doctor'. Disfigured by blaster. Has death sentence in over 12 systems. Notorious partner of Ponda Baba. Aliases: Dr. Cornelius and Rooboo.\n\nText: Adds 2 to power of anything he pilots. The not-so-good doctor may 'operate' on any other character present that was just 'hit' or just Disarmed; 'patient' is lost."
 card "/starwars/ReflectionsII-Dark/t_drevazan&pondababa" "�Dr. Evazan & �Ponda Baba (3)\nDark Character - Alien [PM]\nSet: Reflections II\nPower: 6 Ability: 2\nDeploy: 4 Forfeit: 6\nIcons: Pilot, Warrior\n\nLore: Smugglers who met each other in the Corellian system. Gained the rights to Han Solo's old smuggling routes from Jabba the Hutt. They don't like you either.\n\nText: Adds 3 to power of anything they pilot. Power -3 if opponent has a lightsaber present. Draws one battle destiny if not able to otherwise. May target any other character present that was just 'hit.' Target character is lost."
+card "/starwars/Virtual13-Dark/t_drydenvos" "�Dryden Vos (1)\nDark Character - Alien [C]\nSet: Set 13\nPower: 4 Ability: 3 Teräs Käsi 5\nDeploy: 6 Forfeit: 6\nIcons: Pilot, Warrior\n\nLore: Crimson Dawn leader. Gangster.\n\nText: Deploys -2 if Maul on table. When deployed, may deploy a weapon on Vos from Reserve Deck; reshuffle. While armed and present at a site, opponent may not 'react' to or from here. Immune to attrition < 5 (< 3 if with Qi'ra)."
```

Comment:
`TerÃ¤s KÃ¤si` should be `Teräs Käsi`.
This can be fixed by setting the character set to `windows-1252` and then changing the `Ã¤` to `ä`.
![](https://res.starwarsccg.org/cards/Images-HT/starwars/Virtual13-Dark/large/drydenvos.gif)

Reply:
Those characters render no output in Holotable.  Submitted new PR with just Teras Kasi as the text.